string instruction
uint8 instruction_type
uint8 motor_number
int32 value
---
int32 output
bool result